src/main/java/net/mcreator/ui/modgui/ModElementGUI.java

```diff
@@ -342,6 +342,26 @@ private void disableUnsupportedFields() {
 				}
 			}
 		}
+
+		List<String> exclusions = mcreator.getGeneratorConfiguration()
```

Comment:
I would make special handling where if both inclusions and inclusions are defined, nothing is disabled and warning is shown that both fields can't be used at the same time as it does not make much sense.